src/services/users.js

```diff
@@ -0,0 +1,32 @@
+import { db } from "../firebase-config";
+import {
+  collection,
+  getDocs,
+  getDoc,
+  addDoc,
+  updateDoc,
+  deleteDoc,
+  doc,
+} from "firebase/firestore";
+
+const get = () => getDocs(collection(db, "users"));
+
+const getById = (id) => getDoc(doc(db, "users", id));
+
+const update = async ({ id, user }) => {
+  const userRef = doc(db, "users", id);
+  await updateDoc(userRef, user);
+  return getById(id);
+};
+
+const create = async ({user}) => {
+  const userRef = doc(db, "users");
+  await addDoc(userRef, user);
+  return getById(user);
```

Comment:
`getById` takes a user ID, not a user object. You'll want to try to see if you can get the object you just created, or see if `addDoc` returns that to you